src/index.ts

```diff
@@ -153,7 +153,7 @@ export class KeyValueCache {
         }
       }
     } else {
-      for (const [k, v] of this.#appCache) {
+      for (const [k, v] of Array.from(this.appCache)) {
```

Comment:
I am not sure about this, I just used Array.from to convert appCache to an Array as we need a for here.